src/lib.rs

```diff
@@ -315,59 +321,90 @@ pub fn run(config: Config) -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
-#[derive(Debug, PartialEq)]
-enum CliCommand {
-    Start,
-}
+pub fn parse_args<I, T>(args: I) -> CliArgs
+where
+    I: IntoIterator<Item = T>,
+    T: Into<OsString> + Clone,
+{
+    let app_matches = App::new(clap::crate_name!())
+        .version(clap::crate_version!())
+        .author(clap::crate_authors!())
+        .about(clap::crate_description!())
+        .setting(AppSettings::SubcommandRequiredElseHelp)
+        .subcommand(
+            SubCommand::with_name("start")
+                .about("Start a new tmux session")
```

Comment:
I think it might be helpful to emphasize that the parameterized project config file will be used to start the session.

Riffing on what's used by tmuxinator: "Start a tmux session using a path to a project config file"

What do you think?

Reply:
That's sounds good to me! I'll update it with all the other suggestions.